<launch>
  <node pkg="tf" type="static_transform_publisher" name="camera_fake_localize" args="0.08 -0.05 0.14 0 0 0 base_laser_link orb_camera 20"/>
  <include file="$(find realsense_ros_camera)/launch/rs_rgbd.launch" />
  <node pkg="vo_orb" type="vo_orb_rgbd" name="ORB2_RGBD" args="$(env ORB_PATH)/Vocabulary/ORBvoc.txt $(find vo_orb)/config/d435.yaml" output="screen">
    
    <remap from="/camera/rgb/image_raw" to="/camera/color/image_raw" />
    <remap from="/camera/depth_registered/image_raw" to="/camera/depth/image_rect_raw" />
    
    <!-- <remap from="/camera/rgb/image_raw" to="/camera/color/image_raw" />
    <remap from="/camera/depth_registered/image_raw" to="/camera/depth/image_raw" /> -->
    <param name="cameralink_frame" value="camera_link" />
    <param name="cameraopticallink_frame" value="camera_color_optical_frame" />
  </node>
</launch>

